### ISSUE TITLE
Question about Default Training Settings

### ISSUE THREAD

**Issue by GuanWei926** [jayin92/Lab4-FlowMatching#8]
助教您好：

在 README 中提到的training configuration（如 Batch size: 16、Training steps: 100,000、Learning rate: 2e-4（含 warmup: 200 steps）），我在實際訓練過程中發現模型有些不穩定的情況。為了讓模型能更穩定地收斂，我想調整部分參數。想請問 README 中提供的這些 training configuration 是作為預設參考值，可以進行調整，還是我們需要嚴格依照這些設定進行訓練呢？

感謝助教
